internal/commands/whoami/inputs.go

```diff
@@ -0,0 +1,14 @@
+package whoami
+
+import (
+	"github.com/10gen/realm-cli/internal/cli/user"
+	"github.com/10gen/realm-cli/internal/terminal"
+)
+
+type inputs struct {
+	ShowProjects bool
+}
+
+func (i *inputs) Resolve(profile *user.Profile, ui terminal.UI) error {
```

Comment:
for the no-op `Resolve` function, it's actually not necessary (hooray go and its interfaces!)

also, fwiw, if you _did_ need to implement `Resolve` for something here, it would only ever get called if you also then implemented the `CommandInputs` interface on the `Command` struct itself